src/Pages/Genre.js

```diff
@@ -0,0 +1,24 @@
+import {useState} from 'react';
+import {useParams} from 'react-router-dom';
+import mockData from '../mockdata.json';
+
+export const Genre = () => {
+  let param = useParams().genre;
+
+  param = param[0].toUpperCase() + param.slice(1);
+  let selectedGenre = [];
+  //mockData will be real store data TODO
+  mockData.movies.map((movie) => {
```

Comment:
Let's use state for now `const [movies,setMovies] = useState()
useEffect(()=>{
   setMovies(mockData.filter(movie=>movie.genres.includes(genre)))
},[genre])
`

Reply:
Done